src/main/java/com/adobe/commerce/cif/api/ProductApi.java

```diff
@@ -60,6 +60,21 @@ Product getProductById(
         @HeaderParam(ACCEPT_LANGUAGE) String acceptLanguage
     );
 
+    @GET
+    @Path("/slug/{slug}")
```

Comment:
We always use plural for endpoints like products, carts, so IMO this should be slugs